### PR TITLE
Orders service 

### PR DESCRIPTION
**Pull request by eggwhat**

### CODE REVIEW

**Comment by eggwhat** [comment]
Like I mentioned in #40, the Order will need changes to be able to work without customerId.

**Comment by SaintAngeLs** [comment]
> So far order has only one prop TotalPrice. But it should probably have more details like "_**This page should also show a summary that will contain source and destination information, price split on taxes, service fee, distance fee, and additional fees.**_". We don't have pricing service yet, so it will need to be changed later.

We will update the the more information, later, when we will have to do the mailing, and when the more of the system will be ready

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api.csproj [hunk 1]
```diff
@@ -7,7 +7,17 @@
...
+    <PackageReference Include="Convey.Logging" Version="1.1.448" />
+    <PackageReference Include="Convey.Logging.CQRS" Version="1.1.448" />
+    <PackageReference Include="Convey.Secrets.Vault" Version="1.1.448" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\SwiftParcel.Services.Orders.Infrastructure\SwiftParcel.Services.Orders.Infrastructure\SwiftParcel.Services.Orders.Infrastructure.csproj" />
+  </ItemGroup>
+
```
Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api.csproj:22]: Please, disable nulables

```xml
 <Nullable>enable</Nullable>
```

make it disable

Comment (SaintAngeLs) [reply SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api.csproj:22]: In each of the project

**Review by SaintAngeLs — CHANGES_REQUESTED** [review]
Update the nullability in the project '_..._.csproj' files, please.

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api.csproj (hunk 1 above)
Comment (eggwhat) [reply SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api/SwiftParcel.Services.Orders.Api.csproj:22]: Done

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Exceptions/ExceptionToMessageMapper.cs
Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Exceptions/ExceptionToMessageMapper.cs:1]: Please, resolve this one 
```bash
/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Exceptions/ExceptionToMessageMapper.cs(14,22): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. [/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure.csproj]
```

File: SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Mongo/Documents/Extensions.cs
Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Mongo/Documents/Extensions.cs:1]: Please, would you be so king to resolve this one:

```zsh
/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Extensions.cs(114,43): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. [/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure.csproj]


```

Comment (SaintAngeLs) [inline SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Mongo/Documents/Extensions.cs:1]: Please, try resolving this one

```zsh

/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/Extensions.cs(119,52): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. [/home/kaliuser/Documents/portfolio/commercial_apps/courierapp/courier_app/SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure/SwiftParcel.Services.Orders.Infrastructure.csproj]


```

**Review by SaintAngeLs — CHANGES_REQUESTED** [review]
Please, I would like to ask to to resolve all the nullabilities and nullable references

**Comment by eggwhat** [comment]
Sorry about that, should be ok now

**Review by SaintAngeLs — APPROVED** [review]
Now it is nice, please merge it to development branch

**Pull request merged**